sklearn/dummy.py

```diff
@@ -45,6 +45,10 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
           the user. This is useful for metrics that evaluate a non-majority
           class
 
+          .. versionchanged:: 0.22
+             The default value of `strategy` will change from "stratified" in
+             version 0.22 to "prior" in version 0.24.
```

Comment:
Not sure I understand what you want to say.

We could either just say that it will be changed in 0.24, or also mention that a warning is raised starting from 0.22

Reply:
I am using the same wording in https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html#sklearn.ensemble.RandomForestRegressor

But I see what you mean.